client/authentication/firebase.js

```diff
@@ -18,6 +18,7 @@ export const app = firebase.initializeApp(firebaseConfig);
 
 // auth is used in client/AuthContext.js
 export const auth = app.auth();
+auth.setPersistence(firebase.auth.Auth.Persistence.LOCAL);
```

Comment:
where is set persistence coming from? Is it a firebase function? You don't have to answer.  I'm just thinking out loud.

Reply:
It's [firebase persistence](https://firebase.google.com/docs/auth/web/auth-state-persistence) but I did notice it was local by default, just wanted to make sure no webpack/environment build messes with it